src/clientversion.h

```diff
@@ -44,7 +44,8 @@ static const int CLIENT_VERSION =
 extern const std::string CLIENT_NAME;
 extern const std::string CLIENT_BUILD;
 
-std::string FormatFullVersion();
+std::string FormatVersion(int nVersion);
+std::string FormatVersionAndSuffix();
 std::string FormatSubVersion(const std::string& name, int nClientVersion, const std::vector<std::string>& comments);
```

Comment:
While at it, let's make this clear too - change htis to `FormatUserAgentString` (Also cover rename boost tests accordingly)

Reply:
Okay